clear-containers-image/debian.changelog

```diff
@@ -1,3 +1,25 @@
+clear-containers-image (18400-36) stable; urgency=medium
+
+  * Update clear-containers-image from 18220 to 18400.
+
+  * version: 18280
+  Changes in package iptables (from 1.6.1-20 to 1.6.1-21):
```

Comment:
Nit: I'd personally indent a little more (something like the following for maximum clarity - this is also still valid):

```
 * version: 18280

   Changes in package iptables (from 1.6.1-20 to 1.6.1-21):

       George T Kramer - iptables: Autospec creation for version 1.6.1

   https://download.clearlinux.org/releases/18280/clear/RELEASENOTES

```